locustfile.py

```diff
@@ -0,0 +1,237 @@
+from locust import HttpLocust, TaskSet, task
+import json
+import os,binascii
+
+USER_CREDENTIALS = [
+    ("student1", "student1"),
+    ("student2", "student2"),
+    ("student3", "student3"),
+    ("student4", "student4"),
+    ("student5", "student5"),
+    ("student6", "student6"),
+    ("student7", "student7"),
+    ("student8", "student8"),
+    ("student9", "student9"),
+    ("student10", "student10"),
+    ("student11", "student11"),
+    ("student12", "student12"),
+    ("student13", "student13"),
+    ("student14", "student14"),
+    ("student15", "student15"),
+    ("student16", "student16"),
+    ("student17", "student17"),
+    ("student18", "student18"),
+    ("student19", "student19"),
+    ("student20", "student20"),
+    ("student21", "student21"),
+    ("student22", "student22"),
+    ("student23", "student23"),
+    ("student24", "student24"),
+    ("student25", "student25"),
+    ("student26", "student26"),
+    ("student27", "student27"),
+    ("student28", "student28"),
+    ("student29", "student29"),
+    ("student30", "student30"),
+    ("student1", "student1"),
+    ("student2", "student2"),
+    ("student3", "student3"),
+    ("student4", "student4"),
+    ("student5", "student5"),
+    ("student6", "student6"),
+    ("student7", "student7"),
+    ("student8", "student8"),
+    ("student9", "student9"),
+    ("student10", "student10"),
+    ("student11", "student11"),
+    ("student12", "student12"),
+    ("student13", "student13"),
+    ("student14", "student14"),
+    ("student15", "student15"),
+    ("student16", "student16"),
+    ("student17", "student17"),
+    ("student18", "student18"),
+    ("student19", "student19"),
+    ("student20", "student20"),
+    ("student21", "student21"),
+    ("student22", "student22"),
+    ("student23", "student23"),
+    ("student24", "student24"),
+    ("student25", "student25"),
+    ("student26", "student26"),
+    ("student27", "student27"),
+    ("student28", "student28"),
+    ("student29", "student29"),
+    ("student30", "student30")
+]
+
+class UserBehavior(TaskSet):
+    def on_start(self):
+        """ on_start is called when a Locust start before any task is scheduled """
+        if len(USER_CREDENTIALS) > 0:
+            user, password = USER_CREDENTIALS.pop()
```

Comment:
Is the `UserBehavior` object persistent across users? If so, it would be better to just keep an incrementing index pointing into the user list, so we're not destroying the original data structure.